### ISSUE TITLE
A gift from Chicago

### ISSUE THREAD

**Issue by Cribstone** [mainehackerclub/open_vehicle_tracker#20]
The folks over at Open Chicago have a near perfectly documented solution for us!  First check out- http://clearstreets.org/about
The check out- http://clearstreets.org/about#how-does-it-really-work for a detailed explanation of their process and a list of open source resources I never would've heard about in a million years!  These guys are the best!

**Comment by janewilkin** [comment]
This is awesome.  NYC also had an app for this too, but probably not as well documented:

http://www.nydailynews.com/new-york/plownyc-real-time-snow-cleaning-info-online-article-1.1259230

I'm going to read through their approach in more detail.

**Comment by Cribstone** [comment]
Yes PlowNYC is based on https://github.com/GeoWebCache/geowebcache and www.geoserver.org, while the mobile app uses http://openlayers.org/.  Its basically a component of the NYC-IT GIS platform which was a major victory for open source when they partnered with http://opengeo.org/  for the service contract over ESRI...(see https://github.com/mainehackerclub/open_vehicle_tracker/issues/17 for more on OpenGeo as an Open Source Business Model)
   As far as I know, ESRI has almost complete dominance over municipal GIS systems in Maine so this wouldn't really be an option for us.  The Chicago method is not just much simpler, it uses the same Google Fusion Tables based process I've been looking into for GTFS.  Even better, the only city in Maine to officially deploy real-time transit location data, Portland, selected the same vendor who did the Chicago GTFS-Realtime.  So the even API they produce should be compatible with any template we build from Chicago.

**Comment by janewilkin** [comment]
Yay!

Garrett Wilkin
voice: (207) 370-1862
profile: http://about.me/garrettwilkin
writing: http://www.programmableweb.com/profile/garrettwilkin

**Comment by jqtrde** [comment]
Also:
- http://opentripplanner.com/misc/gtfs/
- https://github.com/UlmApi/livemap
